CHANGELOG.md

```diff
@@ -1,4 +1,31 @@
 ## 4.59.0 (Unreleased)
+
+NOTES:
+* updated google golang api to v0.114.0 to support oauth2 token exchange over mTLS ([#5343](https://github.com/hashicorp/terraform-provider-google-beta/pull/5343))
```

Comment:
I wasn't sure if this was a note we would want to include. I believe most users would not care about this?